modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js

```diff
@@ -462,6 +535,18 @@ export default function ChangeTrackingIndicator({
 										displayType="secondary"
 										onClick={() => {
 											setCloseWarning(true);
+
+											if (popoverCheckbox) {
+												const url = setParameter(
+													saveDisplayPreferenceURL,
+													'silenceContextChangePopover',
+													silenceContextChangePopover
+												);
+
+												fetch(url, {
+													method: 'POST',
+												});
+											}
```

Comment:
When do we store the preference? Is it only when one of the three buttons is clicked? If not, we can do this in the popover's `onShowChange` when the popover is closed.

Reply:
Yes, is is only when we click in one of the buttons